include/aws/mqtt/mqtt.h

```diff
@@ -88,6 +88,9 @@ void aws_mqtt_library_init(struct aws_allocator *allocator);
 AWS_MQTT_API
 void aws_mqtt_library_clean_up(void);
 
+AWS_MQTT_API
+void aws_mqtt_fatal_assert_library_initialized(void);
```

Comment:
Are we going to call this anywhere?

Reply:
🤷‍♂ Nothing has a hard dependency on it yet, but @graebm wanted it present

Reply:
Maybe just shove it in client create and call it good enough for now?

Reply:
Sure #yolo